scalafix-core/shared/src/main/scala/scalafix/internal/rule/RemoveUnusedImports.scala

```diff
@@ -28,6 +28,6 @@ case class RemoveUnusedImports(index: SemanticdbIndex)
   }
   override def fix(ctx: RuleCtx): Patch =
     ctx.tree.collect {
-      case i: Importee if isUnused(i) => ctx.removeImportee(i)
+      case i: Importee if isUnused(i) => ctx.removeImportee(i).atomic
```

Comment:
In the case where there is no `.atomic`, should we implicitly assume `.atomic` on the whole returned `Patch`? This would mean one `// scalafix:ok Blah` disables `Blah` for the whole file, which might be counter intuitive. Alternatively, we can also keep the existing behavior where no `.atomic` means the rule does not respect escape hatches and is applied everywhere regardless of `// scalafix:off blah`. I'm personally leaning towards the second option.